cypress-shared/integration/affirm/2.1-graphql.spec.js

```diff
@@ -0,0 +1,35 @@
+import { updateRetry, loginViaCookies } from '../../support/common/support'
+import {
+  graphql,
+  version,
+  affirmSettings,
+  orderData,
+  orderUpdate,
+  validateGetVersionResponse,
+  validateAffirmSettingsResponse,
+  validateOrderDataResponse,
+  validateOrderUpdateResponse,
+} from '../../support/affirm-payment/graphql_testcase'
```

Comment:
@saravananvenkatesan Move outputvalidation.js and graphql_testcase.js to affirm folder

Reply:
@Syed-Vtex  moved the two files to affirm